packages/iris-grid/src/IrisGridModel.test.ts

```diff
@@ -154,11 +159,13 @@ it('closes the table correctly when the model is closed', () => {
 });
 
 describe('totals table tests', () => {
-  const TOTALS_CONFIG = { operationMap: { 1: ['Max', 'Min'] } };
+  const TOTALS_CONFIG = ({
+    operationMap: { 1: ['Max', 'Min'] },
+  } as unknown) as UITotalsTableConfig;
```

Comment:
Could rewrite to remove the cast:
```suggestion
  const TOTALS_CONFIG: UITotalsTableConfig = {
    operationMap: { 1: ['Max', 'Min'] },
    operationOrder: [],
    showOnTop: true,
  };
```